README.md

```diff
@@ -24,7 +24,7 @@ completely useless in the event that the ASG's update on the launch configuratio
 ## Behavior
 
 On interval, this application:
-1. Iterates over each ASG defined by the `AUTO_SCALING_GROUP_NAMES` environment variable, or each ASG that belong to the cluster if `CLUSTER_NAME` is specified
+1. Iterates over each ASG defined by the `AUTO_SCALING_GROUP_NAMES` environment variable, or each ASG that matches the tags specified in `AUTODISCOVERY_TAGS`.
```

Comment:
Can you clarify the behavior of `CLUSTER_NAME`? (since it's now backward compatible and takes priority over `AUTODISCOVERY_TAGS`)